ruby/ql/src/utils/modeleditor/ModelEditor.qll

```diff
@@ -62,44 +62,44 @@ abstract class Endpoint instanceof DataFlow::Node {
  * A callable method or accessor from source code.
  */
 class MethodEndpoint extends Endpoint {
```

Comment:
Does the following work? If so it should be possible to replace `this.(DataFlow::MethodNode)` with `super`
```suggestion
class MethodEndpoint extends Endpoint instanceof  DataFlow::MethodNode {
```